src/daemon/rpmostreed-deployment-utils.c

```diff
@@ -127,6 +127,24 @@ rpmostreed_deployment_generate_blank_variant (void)
 }
 
 static void
+variant_add_metadata_attribute (GVariantDict *dict,
+                                const gchar  *attribute,
+                                GVariant     *commit)
+{
+  g_autoptr(GVariant) metadata = NULL;
+  g_autofree gchar *attribute_string_value = NULL;
+
+  metadata = g_variant_get_child_value (commit, 0);
+
+  if (metadata != NULL)
+    {
+      g_variant_lookup (metadata, attribute, "s", &attribute_string_value);
+      if (attribute_string_value != NULL)
+        g_variant_dict_insert (dict, attribute, "s", attribute_string_value);
```

Comment:
Hmm, it might be called `ostree.endoflife` in the ostree commit metadata, but is that what we want it to be called in the deployment GVariant too? The key name chosen here shows up also in the output of `rpm-ostree status --json`. Maybe just `end-of-life` would be more appropriate there?